### DISCUSSION TITLE
Enable "GraphiQL Explorer" on Altair (Off-line: no internet connection)

### DISCUSSION THREAD

**Discussion by varasinp** [graphql-dotnet/server#513] [Q&A]
My target release environment does not have an internet connection so I modified Altair UI code to use local CDN files.
(Original) https://github.com/graphql-dotnet/server/tree/master/src/Ui.Altair 
(My forked) https://github.com/varasinp/server/tree/master/src/Ui.Altair

Everything works fine in my release environment except when I try to enable "GraphiQL Explorer" plug-in https://github.com/OneGraph/graphiql-explorer, Altair couldn't find it. I assumed that Altair requires an internet connection to download the plug-in. 

Is there a way for users to use the plug-in without an internet connection? How do I add GraphiQL Explorer?

**Comment by sungam3r** [comment]
I think it's better to ask Altairs's author. Our middleware is just a thin wrapper.

**Reply by varasinp** [reply]
Thank you. I asked the author and he said Altair needs internet access to download the GraphiQL Explorer plug-in.
Here's the question that I posted on Altair Q&A
https://github.com/imolorhe/altair/discussions/1468
